test/components/selectionBoxLayerTests.ts

```diff
@@ -48,6 +48,43 @@ describe("Interactive Components", () => {
         svg.remove();
       });
 
+      it("generates the correct clipPath", () => {
+        sbl.renderTo(svg);
+
+        TestMethods.verifyClipPath(sbl);
+        let clipRect = (<any> sbl)._boxContainer.select(".clip-rect");
+        assert.strictEqual(TestMethods.numAttr(clipRect, "width"), SVG_WIDTH, "the clipRect has an appropriate width");
+        assert.strictEqual(TestMethods.numAttr(clipRect, "height"), SVG_HEIGHT, "the clipRect has an appropriate height");
+        svg.remove();
+      });
```

Comment:
Again, I'm not a big fan of copy + paste-ing. Hence remove this copy from the `dragBoxLayerTests.ts` as we should assume similar behavior